packages/js/e2e-utils-playwright/src/editor.js

```diff
@@ -0,0 +1,164 @@
+const { expect } = require( '@playwright/test' );
+
+export const closeChoosePatternModal = async ( { page } ) => {
+	const closeModal = page
+		.getByLabel( 'Scrollable section' )
+		.filter()
+		.getByRole( 'button', {
+			name: 'Close',
+			exact: true,
+		} );
+	await page.addLocatorHandler( closeModal, async () => {
+		await closeModal.click();
+	} );
+};
+
+export const disableWelcomeModal = async ( { page } ) => {
+	// Further info: https://github.com/woocommerce/woocommerce/pull/45856/
+	await page.waitForLoadState( 'domcontentloaded' );
+
+	const isWelcomeGuideActive = await page.evaluate( () =>
+		wp.data.select( 'core/edit-post' ).isFeatureActive( 'welcomeGuide' )
+	);
+
+	if ( isWelcomeGuideActive ) {
+		await page.evaluate( () =>
+			wp.data.dispatch( 'core/edit-post' ).toggleFeature( 'welcomeGuide' )
+		);
+	}
+};
+
+export const openEditorSettings = async ( { page } ) => {
+	// Open Settings sidebar if closed
+	if ( await page.getByLabel( 'Editor Settings' ).isVisible() ) {
+		console.log( 'Editor Settings is open, skipping action.' );
+	} else {
+		await page.getByLabel( 'Settings', { exact: true } ).click();
+	}
+};
+
+export const getCanvas = async ( page ) => {
+	return page.frame( 'editor-canvas' ) || page;
+};
+
+export const goToPageEditor = async ( { page } ) => {
```

Comment:
This seems like overkill just to load up a new page editor.  I think we can probably eliminate this and simplify things.

Reply:
@lanej0 I've migrated all the utils that are actively used in the tests.

Reply:
Okay, not hurting anything, so we'll leave it as is for now!

Reply:
I think this is acceptable as a util, because it also disabled/closes 2 modals - the reason it exists as a method. I would however rename it to be explicit on its function. Something like `goToPageEditorAndDisableModals` would be better I think.